### PR TITLE
Create zhi.md

### PR DESCRIPTION
**Pull request by kenzliang**
creating a new .md file for the workshop

### CODE REVIEW

File: Participants/zhi.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Hi, I'm [your-github-username]
```
Comment (i-marsh) [inline Participants/zhi.md]: missing github handle 😢 can you fix please

**Review by i-marsh — CHANGES_REQUESTED** [review]

**Comment by kenzliang** [comment]
changes made to add my git username

**Review by nathos — APPROVED** [review]
Looks great! Thanks for making the changes!

emmastone_thumbsup

**Review by i-marsh — APPROVED** [review]

**Comment by i-marsh** [comment]
go ahead and merge then delete your branch 
tenor-163372716

**Pull request closed without merging**